google/resource_sql_ssl_cert.go

```diff
@@ -0,0 +1,177 @@
+package google
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"google.golang.org/api/sqladmin/v1beta4"
+)
+
+func resourceSqlSslCert() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceSqlSslCertCreate,
+		Read:   resourceSqlSslCertRead,
+		Delete: resourceSqlSslCertDelete,
+
+		SchemaVersion: 1,
+
+		Schema: map[string]*schema.Schema{
+			"instance": &schema.Schema{
```

Comment:
We have a particular ordering of schema elements we like to maintain.  First, all `Required` fields, then, all `Optional`s (including `Optional` / `Computed`), then all `Computed`.  Within each group, we like to sort alphabetically.